development/generate-icon-names.js

```diff
@@ -39,8 +39,6 @@ const generateIconNames = async () => {
         getIconNameKebabCase(fileName)),
   );
 
-  console.log('ICON_NAMES env var successfully generated!');
-
```

Comment:
Removing per this convo https://github.com/MetaMask/metamask-extension/pull/15772#discussion_r972369434